packages/admin/resources/js/app.js

```diff
@@ -70,6 +70,29 @@ window
 Chart.defaults.font.family = `'DM Sans', sans-serif`
 Chart.defaults.color = '#6b7280'
 
+const stickyTrigger = document.querySelector('.filament-sticky-trigger')
+const stickyParent = document.querySelector('.filament-main')
+
+if (stickyTrigger) {
+    window.addEventListener('load', function () {
+        const observer = new IntersectionObserver(
+            ([e]) => {
+                if (e.isIntersecting) {
+                    stickyParent.classList.remove('is-sticky')
+                    return
+                }
+
+                stickyParent.classList.add('is-sticky')
+            },
+            {
+                threshold: [0],
+            },
+        )
+
+        observer.observe(stickyTrigger)
+    })
+}
+
```

Comment:
Could this be moved into the header Blade component, @awcodes?

Reply:
Could be moved to the blade component. My reasoning is that it'd be need on the majority of the pages so it'd be better to have it minified as part of the bundle.

Reply:
Hmm I see. Personally, I also like having it in the context it's used instead of the general `app.js` file.